app/actors/hyrax/actors/environment.rb

```diff
@@ -0,0 +1,28 @@
+module Hyrax
+  module Actors
+    class Environment
```

Comment:
Right, this was introduced a couple of months earlier. It was updated recently for rails 5.1, which breaks our blank nodes support. I can't find a better way to fix that so it's customized for our need.